### DISCUSSION TITLE
Is a q-input ready for size changes?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#16059] [General - Components / Directives / etc] [answered]
I use a `q-input` to get important data on a page. I would like it to be much larger than the default one (both font and size). An example code is at https://codepen.io/WoJWoJ/pen/yLQvegO?editors=1011 - I would like the input's font to be, say, three times the size.

I tried to force a larger text (including in the Dev Tools but it either does not work, or the change is not consistent with the rest of the `q-input`.  Typically I can make the font larger but the outline will not follow (it will stay the same).

Is there a standard mechanism to set its size (font & height per the font size)?

**Comment by suleimanas** [comment]
you can use css and target .q-field__control class

```
.q-field__control {
  height: 100px;
  font-size: 50px;
}
```

**Reply by wsw70** [reply]
Unfortunately it does not work (I already tried to play with the CSS) - I think because the component is quite complex and by changing one CSS class the others do not follow through. 

In your example the font is larger and the height of the component is bigger but the interior does not follow (the outline for instance)

**Reply by suleimanas** [reply, marked as answer]
it does work.. but it was just an example, just to give you an idea. add to your codepen

css:
```
.q-field__control {
  display: flex;
  align-items: center;
  height: 80px;
  font-size: 40px;
}
```

and add `class="fit"` to the appended chip component

**Reply by wsw70** [reply]
AH! I removed `scoped` in my `<style>` and it indeed does work! I was not aware that `scoped` not only prevents styles from going "up" but also from going "down".

Thanks a lot for your help!

**Comment by metalsadman** [comment]
ootb is only normal and `dense`. so other customization you will have to tinker with its css to suit your needs. like on a wrapper, re-use it across your app, or override globally.

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by wsw70** [reply]
You are right, sorry, should have started with that. Codepen added to the question
